### PR TITLE
Trigger Rename & Focus for Object, Group, Layout, Events

### PR DESCRIPTION
**Pull request by ClementPasteau**
Focused here and there for better usability

https://user-images.githubusercontent.com/4895034/204303367-656ad46b-d480-4e87-bbcb-6754163741e0.mov

### CODE REVIEW

File: newIDE/app/src/ObjectEditor/ObjectEditorDialog.js [hunk 1]
```diff
@@ -200,6 +200,7 @@ const InnerDialog = (props: InnerDialogProps) => {
                     notifyOfChange();
                   }
                 }}
+                ref={field => field && field.focus()} // Focus the name when the dialog is opened.
```
Comment (AlexandreSi) [inline newIDE/app/src/ObjectEditor/ObjectEditorDialog.js:203]: Should you use the `autofocus` props for that?

File: newIDE/app/src/ProjectCreation/NewProjectSetupDialog.js [hunk 2]
```diff
@@ -165,6 +164,7 @@ const NewProjectSetupDialog = ({
               <Refresh />
             </IconButton>
           }
+          ref={field => field && field.focus()} // Focus the name field when the dialog is opened.
```
Comment (AlexandreSi) [inline newIDE/app/src/ProjectCreation/NewProjectSetupDialog.js:167]: Same here for the autofocus props

File: newIDE/app/src/ProjectCreation/NewProjectSetupDialog.js [hunk 3]
```diff
@@ -130,13 +130,12 @@ const NewProjectSetupDialog = ({
...
           onClick={onClose}
         />,
-        <LeftLoader isLoading={isOpening}>
+        <LeftLoader isLoading={isOpening} key="create">
           <DialogPrimaryButton
             primary
             disabled={
               isOpening || needUserAuthentication || hasTooManyCloudProjects
             }
-            key="create"
```
Comment (AlexandreSi) [inline newIDE/app/src/ProjectCreation/NewProjectSetupDialog.js:139]: 💜

File: newIDE/app/src/ObjectEditor/ObjectEditorDialog.js (hunk 1 above)
Comment (ClementPasteau) [reply newIDE/app/src/ObjectEditor/ObjectEditorDialog.js]: didn't know about this prop! nice!

**Review by 4ian — APPROVED** [review]

**Pull request merged**